lib/libutils/isoc/arch/riscv/setjmp_rv.S

```diff
@@ -0,0 +1,81 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/*
+ * Copyright 2022-2023 NXP
+ */
+#include <asm.S>
+
+#ifdef RV32
+#define STR       sw
+#define LDR       lw
+#define REGOFF(x) ((x) * 4)
+#else
+#define STR       sd
+#define LDR       ld
+#define REGOFF(x) ((x) * 8)
+#endif /*RV32*/
+
+// int setjmp (jmp_buf)
+	.global	setjmp
+	.type	setjmp, %function
+setjmp:
```

Comment:
Please make this:
`FUNC setjmp , :` 
and drop the attributes above.

Same below for `longjmp()`.